src/test/ui/const-generics/min_const_generics/macro.rs

```diff
@@ -0,0 +1,50 @@
+// run-pass
+#![feature(min_const_generics)]
+
+struct Example<const N: usize>;
+
+macro_rules! external_macro {
+  () => {{
+    const X: usize = 1337;
+    X
+  }}
+}
+
+trait Marker<const N: usize> {}
+impl<const N: usize> Marker<N> for Example<N> {}
+
+fn make_marker() -> impl Marker<{
+    #[macro_export]
+    macro_rules! const_macro { () => {{ 3 }} }; inline!()
+}> {
+  Example::<{ const_macro!() }>
+}
+
+fn from_marker(_: impl Marker<{
+    #[macro_export]
+    macro_rules! inline { () => {{ 3 }} }; inline!()
+}>) {}
+
+fn main() {
+  let _ok = Example::<{
+    #[macro_export]
+    macro_rules! gimme_a_const {
+      ($rusty: ident) => {{ let $rusty = 3; *&$rusty }}
+    };
+    gimme_a_const!(run)
+  }>;
```

Comment:
looking at the hackmd it seems like there is one last case we forgot here.

Please define a `#[macro_export]` inside of a repeat expression, so `[0; macro def]` :sweat_smile:

Reply:
I believe there's already one, would it be necessary to add a use of it?